provisioner/aws-manager.js

```diff
@@ -607,7 +577,7 @@ AwsManager.prototype.instancesInRegion = function (region) {
   if (typeof region === 'string') {
     region = [region];
   }
-  return this.__apiState.instances.filter(function (instance) {
+  return this.__apiState.instances.filter(instance => {
     return region.includes(instance.Region);
   });
```

Comment:
If aggressive could be written as:
`return this.__apiState.instances.filter(i => region.includes(i.Region));` and still fit below 80 chars :)